lib/messages.rb

```diff
@@ -0,0 +1,21 @@
+module Messages
+    def start_banner 
+        border = "=====================================\n"
+        empty = "                                     \n                                     \n"
+        title = "         T I C  T A C  T O E         \n"
+
+        return "\n#{border}#{empty}#{title}#{empty}#{border}\n"
```

Comment:
Bit nitpicky, but remember that Ruby returns the last value in a method so declaring `return` here is unnecessary.